rpc/pool.go

```diff
@@ -20,17 +20,16 @@ import (
 	"net"
 	"sync"
 
+	"github.com/CovenantSQL/CovenantSQL/proto"
 	"github.com/pkg/errors"
 	mux "github.com/xtaci/smux"
-
-	"github.com/CovenantSQL/CovenantSQL/proto"
 )
 
+const MaxPhysicalConnection = 10
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported const MaxPhysicalConnection should have comment or be unexported